QueueManagerTest/packages.config

```diff
@@ -1,4 +1,4 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
-  <package id="NUnit" version="3.4.1" targetFramework="net461" />
+  <package id="NUnitTestAdapter.WithFramework" version="2.0.0" targetFramework="net461" />
```

Comment:
I'll remove version 2 and try to run version 3.  I'm sure I will get it working.